client/src/components/Shared/Checkbox/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {
+  CheckboxContainer,
+  Icon,
+  HiddenCheckbox,
+  StyledCheckbox,
+  CheckboxText,
+  StyleSpan,
+} from './checkbox.style';
+
+const Checkbox = ({ name, checked, ...props }) => (
+  <CheckboxText>
```

Comment:
Should the label be around a div? Not common to see a label wrapping a container. Can you explain what you're doing here?